win32ss/user/ntuser/callback.c

```diff
@@ -1272,4 +1272,46 @@ co_IntImmProcessKey(HWND hWnd, HKL hKL, UINT vKey, LPARAM lParam, DWORD dwHotKey
     return ret;
 }
 
+/* Win: ClientImmLoadLayout */
+BOOL APIENTRY co_ClientImmLoadLayout(HKL hKL, PIMEINFOEX pImeInfoEx)
+{
+    BOOL ret;
+    NTSTATUS Status;
+    IMMLOADLAYOUT_CALLBACK_ARGUMENTS Common = { hKL };
+    ULONG ResultLength = sizeof(IMMLOADLAYOUT_CALLBACK_OUTPUT);
+    PVOID ResultPointer = NULL;
+    PIMMLOADLAYOUT_CALLBACK_OUTPUT Output;
+
+    RtlZeroMemory(pImeInfoEx, sizeof(IMEINFOEX));
+
+    UserLeave();
```

Comment:
Why not `UserLeaveCo();` / `UserEnterCo();` ?

Reply:
OK.